beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/handler/ExpiredMetadataHandler.java

```diff
@@ -146,6 +158,30 @@ private boolean cleanupPartition(
     return true;
   }
 
+  private boolean isIcebergTable(CleanerClient client, String databaseName, String tableName) {
+    try {
+      Map<String, String> tableProperties = client.getTableProperties(databaseName, tableName); // fetching table props
+      String tableType = tableProperties != null ? tableProperties.get("table_type") : null;
+      String format = tableProperties != null ? tableProperties.get("format") : null;
+
+      // Check if tableType or format contain "iceberg"
+      boolean isIcebergByType = tableType != null && tableType.toLowerCase(Locale.ROOT).contains("iceberg");
```

Comment:
Why do you need to use `Locale.ROOT`? Isn't `.toLowerCase()` enough in this case?

Reply:
yes not needed, updated.